app.js

```diff
@@ -297,7 +297,7 @@ function ClientOptionsHandler(fileSources) {
         localStoragePrefix: ceProps('localStoragePrefix'),
         cvCompilerCountMax: ceProps('cvCompilerCountMax', 6),
         defaultFontScale: ceProps('defaultFontScale', 1.0),
-        doCache: defArgs.doCache
+        doCache: defArgs.doCache,
```

Comment:
Super minor: Interesting that the linter did not complain about this comma. Can you remove it, please?

Reply:
Will do!